orderedmap_test.go

```diff
@@ -102,17 +102,40 @@ func TestBasicFeatures(t *testing.T) {
 
 	// check iterations again
 	i = 0
-	for pair := om.Oldest(); pair != nil; pair = pair.Next() {
+	for pair := om.First(); pair != nil; pair = pair.Next() {
 		assert.Equal(t, i, pair.Key)
 		assert.Equal(t, 4*i, pair.Value)
 		i += 2
 	}
 	i = 2 * ((n - 1) / 2)
-	for pair := om.Newest(); pair != nil; pair = pair.Prev() {
+	for pair := om.Last(); pair != nil; pair = pair.Prev() {
 		assert.Equal(t, i, pair.Key)
 		assert.Equal(t, 4*i, pair.Value)
 		i -= 2
 	}
+
+	// check copying
+	com := om.Copy()
+	for np, cp := om.Front(), com.Front(); np != nil; np, cp = np.Next(), cp.Next() {
+		assert.Equal(t, np.Key, cp.Key)
+		assert.Equal(t, np.Value, cp.Value)
+	}
```

Comment:
This is missing a crucial part of the test: that the copy is in fact different from the original. As it stands now
```
func (om *OrderedMap[K, V]) Copy() *OrderedMap[K, V] {
  return om
}
```
would pass these tests.